### DISCUSSION TITLE
Calibre 6.8 on Mac silicone crashes

### DISCUSSION THREAD

**Discussion by trklesch** [apprenticeharper/DeDRM_tools#2215] [Q&A]
I haven't had this problem on Ubunu or Windows 10.  DeDrm 7.21 installs ok but when I try to edit it, it crashes.  I Hope this is fixable

**Comment by ElleKayEm** [comment]
If you are using calibre 6, you need DeDRM 10.0.3: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.3

**Comment by trklesch** [comment]
Thank you for the update

On Mon, Mar 6, 2023 at 15:31 ElleKayEm ***@***.***> wrote:

> If you are using calibre 6, you need DeDRM 10.0.3:
> https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.3
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apprenticeharper/DeDRM_tools/discussions/2215#discussioncomment-5221499>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6I3KL64J2MXUXG4AQT7VIDW2ZXTXANCNFSM6AAAAAAVRXSFB4>
> .
> You are receiving this because you authored the thread.Message ID:
> <apprenticeharper/DeDRM_tools/repo-discussions/2215/comments/5221499@
> github.com>
>
-- 
Sent from Eternity Road
